mozilla-release/mobile/android/base/java/org/mozilla/gecko/BrowserApp.java

```diff
@@ -3359,6 +3379,9 @@ private void showFirstrunPager() {
             mCliqzIntroPagerHolder = findViewById(R.id.cliqz_intro_pager_holder);
             mCliqzIntroPagerHolder.setVisibility(View.VISIBLE);
 
+            final View ghosterySplashScreen = findViewById(R.id.ghostery_splash_screen);
```

Comment:
findViewById(R.id.ghostery_splash_screen) is used multiple times in same class. Reduce it to just one call

Reply:
@ravjit-cliqz done 👍